mosquitto/README.md

```diff
@@ -0,0 +1,101 @@
+# Hass.io Core Add-on: Mosquitto broker
+
+Check your current configuration against any Home Assistant version.
+
+![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+
+## About
+
+You can use this add-on to to install Eclipse Mosquitto which is an open source (EPL/EDL licensed) message broker that implements the MQTT protocol. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers.
+
+## Installation
+
+The installation of this add-on is straightforward and easy to do.
+
+1. Navigate in your Home Assistant frontend to **Hass.io** -> **Add-on Store**.
+2. Find the "Mosquitto broker" add-on and click it.
+3. Click on the "INSTALL" button.
+
```

Comment:
@frenck should we maintain that this add-on use integrations aka config flow?

Reply:
That is a good suggestion. Adding additional steps on how to set it up from the Home Assistant side might help users installing it for the first time.

Reply:
Done, see https://github.com/home-assistant/hassio-addons/pull/748/commits/be1264216a48e5a4c3afdf0e60ff5dbcca0af5d5